tests/CompleteCircuitBreakerTest.php

```diff
@@ -26,10 +26,12 @@ public function setUp() {
 	public function registerRequests(array $requests){
 		foreach ($requests as $time => $success) {
 			$this->timeProvider->set($time);
-			if ($success) {
+			if ($success==TRUE) {
```

Comment:
This doesn't seem quite right. If `$success` must be boolean `TRUE`, compare with `===`. Otherwise, we don't need the comparison.